csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
I THINK if you add a project.json file next to Google.Protobuf.csproj, that will break the old non-dotnet solution (at least that was the case in preview1).  Adding a dummy Google.Protobuf/Google.Protobuf.project.json fixes that.